src/warnet/cli/cluster.py

```diff
@@ -95,7 +91,7 @@ def deploy():
     kubectl apply -f {MANIFEST_PATH}/namespace.yaml
     kubectl apply -f {MANIFEST_PATH}/rbac-config.yaml
     kubectl apply -f {MANIFEST_PATH}/warnet-rpc-service.yaml
-    kubectl apply -f {MANIFEST_PATH}/warnet-rpc-statefulset-dev.yaml
+    sed "s|bitcoindevproject/warnet-rpc:latest|bitcoindevproject/warnet-rpc:{VERSION}|" {MANIFEST_PATH}/warnet-rpc-statefulset.yaml | kubectl apply -f -
```

Comment:
299122363f350ad87b115ca84936eece37d08bd3

making me wonder again if python-embedded-bash (plus external yaml files) can be simplified with just... python.

https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/AppsV1Api.md#create_namespaced_stateful_set

Reply:
or maybe `warnet-rpc-statefulset.yaml` should just always have its own version hard-coded? I'm trying to think if we'd ever need `latest` in prod?